code/modules/projectiles/wavegun.dm

```diff
@@ -75,15 +75,18 @@ toxic - poisons
 			for(var/turf/tile in range(1,T))
 				for(var/atom/movable/O in tile.contents)
 					O.emp_act()
+					SEND_SIGNAL(O, COMSIG_MOVABLE_EMP_ACT)
```

Comment:
Not that important now but I think these could be in the opposite order with the signal being able to use the RETURN_EARLY thing to skip emp_act itself. Opens up a possibility for EMP-shielding by components.

Reply:
Added.